CHANGELOG.md

```diff
@@ -4,6 +4,12 @@ All notable changes to this project will be documented in this file.
 
 The format is loosely based on [Keep a Changelog](http://keepachangelog.com/en/1.0.0/).
 
+## Unreleased
+
+#### Added
+
+* New method overloads `ISetupSequentialResult.PassAsync()` and `ISetupSequentialResult.ThrowsAsync(...)` for async methods with void return (@fuzzybair, #993)
```

Comment:
Consider simplifying this description to:

> _New `SetupSequence` verbs `.PassAsync()` and `.ThrowsAsync(...)` for async methods with `void` return type (@fuzzybair, #993)_

Because (I suspect) the target audience of the changelog&mdash;other Moq users&mdash;will more easily recognise what this is about; `ISetupSequentialResult` isn't something one typically sees mentioned in user code so few people will recognise it.